src/audio_core/cubeb_input.cpp

```diff
@@ -23,7 +23,8 @@ struct CubebInput::Impl {
     static void StateCallback(cubeb_stream* stream, void* user_data, cubeb_state state);
 };
 
-CubebInput::CubebInput() : impl(std::make_unique<Impl>()) {
+CubebInput::CubebInput(std::string_view device_id)
```

Comment:
```suggestion
CubebInput::CubebInput(std::string device_id)
```

and std::move it into the class member.